common/src/main/java/com/joyent/http/signature/KeyPairLoader.java

```diff
@@ -150,36 +179,69 @@ public static KeyPair getKeyPair(final byte[] pKeyBytes, final char[] password)
     /**
      * Read KeyPair from an input stream, optionally using password.
      *
-     * @param is private key content as a stream
+     * @param is       private key content as a stream
      * @param password password associated with key
      * @return public/private keypair object
      * @throws IOException If unable to read the private key from the string
      */
     public static KeyPair getKeyPair(final InputStream is,
-                              final char[] password) throws IOException {
+                                     final char[] password) throws IOException {
+        return getKeyPair(is, password, null);
+    }
+
+    /**
+     * Read KeyPair from an input stream, optionally using password.
```

Comment:
I think a little guidance here would help future users.  For example, if explicit providers are typically only used for testing.